pages/common/ftp.md

```diff
@@ -0,0 +1,23 @@
+# ftp
+
+> Internet File Transfer Program
```

Comment:
Yeah it does, but I guess the program subverts the acronym? I copied and pasted this from the FTP `man` command:
```
NAME
     ftp -- Internet file transfer program
```
But I agree that it's a bit confusing. I'll change this.